kalite/distributed/templates/distributed/base_manage.html

```diff
@@ -14,13 +14,13 @@
         <ul class="nav nav-tabs">
             <div class="mobile-manage-nav">
             {% if request.session.facility_user.facility.id %}
-                <li class="teacher-only users {% block user_active %}{% endblock user_active %}"><a href="{% url 'facility_management' zone_id=None facility_id=request.session.facility_user.facility.id %}" title="{% trans 'Manage Users' %}"><div><span class="icon icon-uniE606"></span><br/>{% trans "Users" %}</div></a></li>
+                <li class="teacher-only users {% block user_active %}{% endblock user_active %}"><a href="{% url 'facility_management' zone_id=None facility_id=request.session.facility_user.facility.id %}" title="{% trans 'Manage Users' %}"><div><i class="icon icon-uniE606"></i><br/>{% trans "Users" %}</div></a></li>
```

Comment:
Don't use spans for icons (got highlighted by the nav tab styling changes above).